.github/workflows/release-java-sdk-to-maven-central.yml

```diff
@@ -53,10 +48,10 @@ jobs:
           java-version: '11'
           distribution: 'temurin'
           cache: 'gradle'
-      - name: Validate Gradle wrapper
-        uses: gradle/wrapper-validation-action@f9c9c575b8b21b6485636a91ffecd10e558c62f6
```

Comment:
gradle/wrapper-valiadtion-action validates gradlew that is not in use in this repo; removing.